csrc/preseg_passes/consecutive_cast.cpp

```diff
@@ -92,7 +189,92 @@ Val* replaceInputInCast(Val* cast_output, Val* new_input) {
 //
 //        b. otherwise, we can't bypass `lo_anchor` cast, we rewire this
 //        section as `starting_anchor`->`lo_anchor`->`expr->output(0)`
+Expr* moveChainedCasts(Expr* expr, std::unordered_set<Expr*>& visited) {
```

Comment:
There's no code change here. Just mechanically moving this into a separate function.